src/filepath.c

```diff
@@ -827,6 +827,9 @@ f_chdir(typval_T *argvars, typval_T *rettv)
 	    (void) check_for_string_arg(argvars, 0);
 	return;
     }
+    if (in_vim9script()
```

Comment:
As this is a new argument, there is no need to check for in_vim9script().  The argument type check can be applied to both legacy scripts and Vim9 scripts.

Reply:
I was copying other parts without fully understanding the meaning behind `in_vim9script()`. Thank you for explaining this to me.